src/explore-education-statistics-frontend/src/services/applicationInsightsService.ts

```diff
@@ -0,0 +1,21 @@
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+
+// eslint-disable-next-line import/prefer-default-export
+export const initApplicationInsights = () => {
+  if (
+    process.env.AI_INSTRUMENTATION !== undefined &&
+    process.env.AI_TRACKING === 'true'
+  ) {
+    const appInsights = new ApplicationInsights({
+      config: {
+        instrumentationKey: '19746be1-8feb-49c2-8c61-3a671628f6e3',
```

Comment:
Need to replace this line with the `process.env.AI_INSTRUMENTATION` variable

Reply:
Do we need to generate a new key @jamescutts

Reply:
Oops yeah added it in